sys/fs/fuse/fuse_vnops.c

```diff
@@ -3208,7 +3214,7 @@ fuse_vnop_vptofh(struct vop_vptofh_args *ap)
 		/* NFS requires lookups for "." and ".." */
 		SDT_PROBE2(fusefs, , vnops, trace, 1,
 			"VOP_VPTOFH without FUSE_EXPORT_SUPPORT");
-		return EOPNOTSUPP;
+		return (EXTERROR(EOPNOTSUPP, "VPTOFH requires FUSE_EXPORT_SUPPORT"));
```

Comment:
"VPTOFH" is a VFS term that will mean nothing to most userland programmers.  Maybe "getfh requires FUSE_EXPORT_SUPPORT" or "Server is missing FUSE_EXPORT_SUPPORT"?